src/rpc/misc.cpp

```diff
@@ -251,14 +252,17 @@ static RPCHelpMan deriveaddresses()
                 }
             },
             RPCExamples{
-                "First three native segwit receive addresses\n" +
+                "First three native segwit receive addresses:\n" +
                 HelpExampleCli("deriveaddresses", "\"" + EXAMPLE_DESCRIPTOR + "\" \"[0,2]\"") +
-                HelpExampleRpc("deriveaddresses", "\"" + EXAMPLE_DESCRIPTOR + "\", \"[0,2]\"")
+                HelpExampleRpc("deriveaddresses", "\"" + EXAMPLE_DESCRIPTOR + "\", \"[0,2]\"") +
+                "Derive all addresses from a privkey cPsQTSmMZ8e3AEUWGjS73f5R364yJxH6RxcgnwbHjbKbFPUP2Dtu:\n" +
```

Comment:
"Derive all addresses from a WIF, which has built-in a checksum:\n"

Reply:
"a WIF privkey" seems more correct to me, but going with your suggestion.